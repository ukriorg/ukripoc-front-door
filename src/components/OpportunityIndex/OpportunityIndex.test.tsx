import React from "react";
import renderer from "react-test-renderer";
import { OpportunityIndex } from "./index";

describe("OpportunityIndex", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<OpportunityIndex opportunities={[]} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
