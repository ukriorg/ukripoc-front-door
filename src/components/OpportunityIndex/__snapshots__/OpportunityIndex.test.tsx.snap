// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpportunityIndex renders correctly 1`] = `<div />`;
